data/family_release/RedHat-7.yaml

```diff
@@ -0,0 +1,14 @@
+---
+svckill::ignore_defaults:
+  - autovt
+  - goferd
+  - ^systemd-.*
```

Comment:
the systemd one yes, there are 3 or 4 systemd services, and service templates in play on rhel7